### ISSUE TITLE
Is it necessary to preprocess the data of the test set?

### ISSUE THREAD

**Issue by sdwiodhi** [MIC-DKFZ/nnUNet#2922]
Hello, I'd like to ask a question. After the training is completed, I need to perform inference on the test set. So, does the data of the test set need to be preprocessed in the same way as the training set and validation set?
